pkg/clients/coredata/request.go

```diff
@@ -0,0 +1,191 @@
+/*******************************************************************************
+ * Copyright 2017 Dell Inc.
```

Comment:
@jpwhitemn How should the copyright work here? This is a new file created by @jduranf but AFAIK he's working on the project himself and his employer isn't a member company. Does that mean he should assign copyright to himself? If his employer IS a member company then their name should be here, correct?

Reply:
Yes - Joan, just add your own copyright below the Dell one.  See https://github.com/edgexfoundry/device-sdk-go/blob/master/config.go as an example.

Something like this:

// -*- mode: Go; indent-tabs-mode: t -*-
//
// Copyright (C) 2017 Dell Inc.
// Copyright (C) 2018 Joan Duran
//
// SPDX-License-Identifier: Apache-2.0